src/test/java/seedu/address/logic/commands/FindCommandTest.java

```diff
@@ -28,36 +29,153 @@ public class FindCommandTest {
     private Model expectedModel = new ModelManager(getTypicalAddressBook(), new UserPrefs());
 
     @Test
-    public void equals() {
-        NameContainsKeywordsPredicate firstPredicate =
-                new NameContainsKeywordsPredicate(Collections.singletonList("first"));
-        NameContainsKeywordsPredicate secondPredicate =
-                new NameContainsKeywordsPredicate(Collections.singletonList("second"));
+    public void testEquals() {
+        List<String> firstSingletonList = Collections.singletonList("first");
+        List<String> secondSingletonList = Collections.singletonList("second");
+        PersonContainsKeywordsPredicate firstPersonPredicate =
```

Comment:
Perhaps consider standardising the formatting here, i.e., the place at which you line break and the subsequent indentation, to improve the readability.

Same concern is relevant in `PersonContainsKeywordPredicateTest.java`